bin/doctrine-migrations.php

```diff
@@ -43,7 +47,7 @@
 $helperSet = ($helperSet) ?: new \Symfony\Component\Console\Helper\HelperSet();
 $helperSet->set(new \Symfony\Component\Console\Helper\DialogHelper(), 'dialog');
 
-$cli = new \Symfony\Component\Console\Application('Doctrine Migrations', \Doctrine\DBAL\Migrations\MigrationsVersion::VERSION);
+$cli = new \Symfony\Component\Console\Application('Doctrine Migrations', '@git-version@');
```

Comment:
Yeah.
I was hoping to find a way to make it work but then I need to make sure that the git repo is there and that can't be guaranteed.